torch/csrc/dynamo/eval_frame.c

```diff
@@ -524,6 +524,25 @@ static PyObject* _custom_eval_frame_shim(
 static PyObject* skip_code_recursive_flag;
 static PyObject* cache_limit_hit_flag;
 
+
+// When this flag is turned on, cache lookup runs diff guard set to find the
+// compiled graph.
+static PyObject* run_diff_guard_set;
+
+static PyObject* set_run_diff_guard_set_flag_py(PyObject* dummy, PyObject* value) {
+  if (value != Py_True && value != Py_False) {
+    DEBUG_TRACE0("arg error");
+    PyErr_SetString(PyExc_TypeError, "expected True/False");
+    return NULL;
+  }
+  run_diff_guard_set = value;
+  Py_RETURN_NONE;
+}
+
+static PyObject* get_run_diff_guard_set_flag_py(PyObject* dummy, PyObject* args) {
+  return run_diff_guard_set;
```

Comment:
Missing incref.  This one is actually dangerous because if you run this code enough times the refcount of True/False will reach zero.